electrumsv/daemon.py

```diff
@@ -312,7 +330,19 @@ def run_cmdline(self, config_options: dict) -> Any:
         result = func(*args, **kwargs)
         return result
 
+    def on_stop(self):
+        if self.rest_server and self.rest_server.is_alive:
+            app_state.async_.spawn_and_wait(self.rest_server.stop)
+        self.logger.debug("stopped.")
+
+    def launch_restapi(self):
+        if not self.is_gui and not self.rest_server.is_alive:
+            self._restapi_future = app_state.async_.spawn(self.rest_server.launcher)
+            self.rest_server.is_alive = True
+
     def run(self) -> None:
+        if app_state.gui_kind != "gui":
```

Comment:
What is `gui_kind == "gui"`? Do you mean `"qt"`? The API should be able to run alongside the GUI in any case.

Reply:
As per discussion I will make the built-in restapi manually activated via .\electrum-sv --restapi
on commandline and remove the logic like `gui_kind == "gui"` for disabling restapi when running the gui